src/modules/vtol_att_control/standard.cpp

```diff
@@ -368,6 +368,12 @@ void Standard::fill_actuator_outputs()
 
 	_actuators_out_1->control[actuator_controls_s::INDEX_THROTTLE] = _pusher_throttle;
 
+	/* translate MC actuators to FW actuators */
+	if (_vtol_schedule.flight_mode == TRANSITION_TO_FW || _vtol_schedule.flight_mode == TRANSITION_TO_MC) {
+		_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = -_actuators_mc_in->control[actuator_controls_s::INDEX_YAW];
+		_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] = _actuators_mc_in->control[actuator_controls_s::INDEX_THROTTLE] - 0.5f;
```

Comment:
I'm not talking about a fixed trim. My concerns:

a) Your patch disregards the current commanded pitch. Has to be fixed.
b) I'm not 100% convinced that the MC thrust is the proper scaling factor but I can see the connection. However, can you verify, or explain why, that the pitch trim you need changes from positive to negative crossing over 0.5 thrust?

On any plane, if the CG changes, you need to adjust trims (either manually or dynamically), but that's not a reason against a trim setting.

Here's what I would try instead:

```
_actuators_out_1->control[actuator_controls_s::INDEX_PITCH] += _actuators_mc_in->control[actuator_controls_s::INDEX_THROTTLE] * _parameters->fw_pitch_trim_trans;
```

New parameter, adding trim in just one direction but still coupled to MC thrust.